packages/components/src/AppLoader/constant.js

```diff
@@ -170,7 +170,7 @@ body {
   }
 }`;
 
-const APP_LOADER = `<div class="tc-app-loader-container">
+const APP_LOADER = `<div class="tc-app-loader-container" aria-label="Loading application" role="status">
```

Comment:
Shouldn't this label be translated ?

Reply:
Not the constant one. The app is not loaded :/